server/py/framework/db/sqldb/models.py

```diff
@@ -861,7 +861,7 @@ class AlertActivation(Base, mlrun.utils.db.BaseModel):
             ),
         )
 
-        id = Column(Integer)
+        id = Column(Integer, autoincrement=True)
```

Comment:
This was already handled in a migration. Won't re-adding this cause conflicts?

Reply:
@TomerShor the problem is that without this string, flush works weird and overall this string just shows to ORM that the row is autoincrementable. Also, we saw the warning in the logs because of this. So, this approach is more correct in terms of working with ORM. 

> Won't re-adding this cause conflicts?

No, it's completely alright